src/cmd_sym.ml

```diff
@@ -104,26 +109,45 @@ let symbolic_extern_module : Symbolic.P.extern_func Link.extern_module =
   { functions }
 
 let summaries_extern_module : Symbolic.P.extern_func Link.extern_module =
+  let open Expr in
+  let i32 v = match v.e with Val (Num (I32 v)) -> v | _ -> assert false in
+  let ptr v = match v.e with Ptr (b, _) -> b | _ -> assert false in
```

Comment:
In both cases, could you use `Log.err` and add a proper error message instead of the `assert false` ?